flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+source "$(dirname "$0")"/common.sh
+
+DOCKER_MODULE_DIR=${END_TO_END_DIR}/../flink-container/docker
+KUBERNETES_MODULE_DIR=${END_TO_END_DIR}/../flink-container/kubernetes
+CONTAINER_SCRIPTS=${END_TO_END_DIR}/test-scripts/container-scripts
+MINIKUBE_START_RETRIES=3
+MINIKUBE_START_BACKOFF=5
+
+function check_kubernetes_status {
```

Comment:
nit: maybe rename it to `get_kubernetes_status` or `is_kubernetes_running`? The `check` seems reflect `void` semantic. But this is not mandatory. You can make a final decision.

Reply:
Make sense. I will update it the `get_kubernetes_status`.

Reply:
I suggest to keep the `check_kubernetes_status`. Because we do not directly get the return value from the shell function. We use `$?` to get the return value.
Adding `return` in a shell function does not always mean we will get return value when calling it.